packages/core/src/components/QueryBuilder/RuleGroup/RuleGroup.tsx

```diff
@@ -0,0 +1,337 @@
+import { useCallback, useContext } from "react";
+import { Add, Delete, Info } from "@hitachivantara/uikit-react-icons";
+import {
+  HvButton,
+  HvEmptyState,
+  HvGrid,
+  HvMultiButton,
+  HvTypography,
+} from "@core/components";
+import { Rule } from "../Rule";
+import { withTooltip } from "@core/hocs";
+import queryBuilderClasses, {
+  HvQueryBuilderClasses,
+} from "../queryBuilderClasses";
+import { QueryBuilderContext } from "../Context";
+import { ClassNames } from "@emotion/react";
+import { styles } from "./RuleGroup.styles";
+
+export interface RuleGroupProps {
+  /**
+   * Override or extend the styles applied to the component.
+   * See CSS API tab for more details.
+   */
+  classes?: HvQueryBuilderClasses;
+  id?: number;
+  level?: number;
+  combinator?: string;
+  rules?: any[];
+}
+
+export const RuleGroup = ({
+  level = 0,
+  id,
+  combinator = "and",
+  rules = [],
+  classes,
+}: RuleGroupProps) => {
+  const context = useContext(QueryBuilderContext);
+
+  const { dispatchAction, askAction, maxDepth, combinators, labels, readOnly } =
+    context;
+  const normalizedMaxDepth = maxDepth - 1;
+
+  const actionButtons = (
+    <ClassNames>
+      {({ css, cx }) => (
+        <>
+          <div
+            className={cx(
+              queryBuilderClasses.buttonBackground,
+              css(styles.buttonBackground, classes?.buttonBackground)
+            )}
+          >
+            <HvButton
+              variant="secondarySubtle"
+              onClick={() => {
+                dispatchAction({ type: "add-rule", id });
+              }}
+              disabled={readOnly}
+              startIcon={<Add />}
+            >
+              {level === 0 && labels.query?.addRule?.label != null
+                ? labels.query?.addRule?.label
+                : labels.group.addRule.label}
+            </HvButton>
+          </div>
+          {level <= normalizedMaxDepth && (
+            <div
+              className={cx(
+                queryBuilderClasses.buttonBackground,
+                css(styles.buttonBackground, classes?.buttonBackground)
+              )}
+            >
```

Comment:
Should be:
cx(queryBuilderClasses.buttonBackground, css(styles.buttonBackground), classes?.buttonBackground)